src/app/public/modules/ag-grid/cell-renderers/cell-renderer-row-selector/cell-renderer-row-selector.component.scss

```diff
@@ -1,5 +1,5 @@
-sky-checkbox {
-  position: absolute;
-  top: -6px;
-  margin: 0 auto;
+::ng-deep {
```

Comment:
This is also another "disable view encapsulation" opportunity. Also, the use of `!important` leads me to believe you should probably be using a more specific selector.